handsontable/src/plugins/nestedHeaders/nestedHeaders.js

```diff
@@ -789,6 +790,19 @@ export class NestedHeaders extends BasePlugin {
     return label;
   }
 
+  /**
+   * `modifyFocusedElement` hook callback.
+   *
+   * @param {number} row Row index.
+   * @param {number} column Column index.
+   * @returns {HTMLTableCellElement} The `TH` element to be focused.
+   */
+  onModifyFocusedElement(row, column) {
```

Comment:
Use the private method here or add `@private` JSDocs tag. Otherwise, the method will be visible in the Docs.

Reply:
Done in https://github.com/handsontable/handsontable/pull/10342/commits/cd04737fe2df87362f40b2f431943dd173226189